config/nova-tinymce-field.php

```diff
@@ -11,7 +11,7 @@
         'init' => [
             'allow_html_in_named_anchor' => false,
             'branding' => false,
-            'extended_valid_elements' => 'a[href]',
+            'extended_valid_elements' => 'a[href],img[*]',
```

Comment:
While we're at it, want to enable everything on anchor tags, too? There are a lot of attributes like "target" and "rel" that are useful to add.

Reply:
So, I agree 100% on target/rel. The reason I did this is because when I was pasting links from Word, it kept adding this weird `name="asdf1234hashxD"` which for some reason rendered a weird icon on the left of all links.

Could we maybe compile a list of allowed attributes on <a> tags?

I would think these to start:
1. `href`
2. `rel`
3. `target`

Anything else come to mind @elliottregan ?

Reply:
Ah, good catch.

How about `data-*`?

Reply:
Sounds good 👍 

I confirmed locally that this works: `a[data-*|href|rel|target]` by manually adding all of those attributes and some others like `random` that I wanted to be deleted in the `code view`. All of the expected attributes were kept, and `random` was deleted 🐒